tests/adapters/switches/dell_test.py

```diff
@@ -265,6 +280,26 @@ def test_get_vlan_with_name(self):
         assert_that(vlan.name, equal_to("shizzle"))
         assert_that(len(vlan.ips), equal_to(0))
 
+    def test_get_vlan_with_line_break(self):
+        self.mocked_ssh_client.should_receive("do").with_args("show vlan id 1000", wait_for=("--More-- or (q)uit", "#"), include_last_line=True).once().ordered().and_return([
+            "VLAN       Name                         Ports          Type      Authorization",
+            "-----  ---------------                  -------------  -----     -------------",
+            "1000   shizzle                          ch2-3,ch5-6,   Static    Required",
+            "                                        ch8-18,",
+            "                                        1/g2-1/g3",
+            "                                                        ",
```

Comment:
This is not how line breaks works, as stated above, empty lines doesn't exist  check the test test_get_vlan_with_more_than_one_page  for an example of a line break